specutils/manipulation/extract_spectral_region.py

```diff
@@ -99,12 +99,16 @@ def extract_region(spectrum, region):
 
     Parameters
     ----------
-    spectrum: `~specutils.spectra.spectrum1d.Spectrum1D`
+    spectrum: `~specutils.Spectrum1D`
         The spectrum object from which the region will be extracted.
 
+    region: `~specutils.SpectralRegion`
+        The spectral region to extract from the original spectrum.
+
     Returns
     -------
-    spectrum: `~specutils.spectra.spectrum1d.Spectrum1D`
+    spectrum: `~specutils.Spectrum1D` or list of `~specutils.Spectrum1D`
+              if input region contains multiple subregions
         Excised spectrum.
```

Comment:
```suggestion
    spectrum: `~specutils.Spectrum1D` or list of `~specutils.Spectrum1D`
        Excised spectrum, or list of spectra if the input region contained multiple subregions.
```

I think it was actually this line break causing problems, we'll see shortly...

Reply:
Yes, I think so, looking at the correct line no. now.
Unfortunately the download error got in the way, but I tested the doc build locally and did not see the warning anymore after fixing the indentation.